seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonError.java

```diff
@@ -87,6 +90,14 @@ public static SeaTunnelRuntimeException writeSeaTunnelRowFailed(
         return new SeaTunnelRuntimeException(WRITE_SEATUNNEL_ROW_ERROR, params, cause);
     }
 
+    public static SeaTunnelRuntimeException unsupportedDataType(
+            String identifier, String dataType) {
+        Map<String, String> params = new HashMap<>();
+        params.put("identifier", identifier);
+        params.put("dataType", dataType);
+        return new SeaTunnelRuntimeException(UNSUPPORTED_DATA_TYPE_SIMPLE, params);
+    }
```

Comment:
Please reuse `public static SeaTunnelRuntimeException unsupportedDataType(
            String identifier, String dataType, String field) `. Because we should make sure user know which field can not supported.